pkg/domain/infra/tunnel/generate.go

```diff
@@ -8,14 +8,11 @@ import (
 )
 
 func (ic *ContainerEngine) GenerateSystemd(ctx context.Context, nameOrID string, opts entities.GenerateSystemdOptions) (*entities.GenerateSystemdReport, error) {
-	options := new(generate.SystemdOptions).WithUseName(opts.Name).WithContainerPrefix(opts.ContainerPrefix).WithNew(opts.New).WithNoHeader(opts.NoHeader).WithTemplateUnitFile(opts.TemplateUnitFile)
+	options := new(generate.SystemdOptions).WithUseName(opts.Name).WithContainerPrefix(opts.ContainerPrefix).WithNew(opts.New).WithNoHeader(opts.NoHeader).WithTemplateUnitFile(opts.TemplateUnitFile).WithStartTimeout(opts.StartTimeout).WithStopTimeout(opts.StopTimeout)
 	options.WithPodPrefix(opts.PodPrefix).WithSeparator(opts.Separator)
 	if opts.RestartPolicy != nil {
 		options.WithRestartPolicy(*opts.RestartPolicy)
 	}
-	if to := opts.StopTimeout; to != nil {
-		options.WithStopTimeout(*opts.StopTimeout)
-	}
```

Comment:
Why did you change this. Now the client will send the default value form containers.conf and it will not use the servers default.

Reply:
Good catch.  I'll fix that.